Kernel_23/doc/Kernel_23/Concepts/FunctionObjectConcepts.h

```diff
@@ -5964,13 +5964,22 @@ class ConstructPoint_2 {
   /// A model of this concept must provide:
   /// @{
 
-
   /*!
     introduces a variable with Cartesian coordinates
     \f$ (0,0)\f$.
   */
   Kernel::Point_2 operator()(const CGAL::Origin &CGAL::ORIGIN);
 
+  /*!
+    returns `p`.
+
+    \note It is advised to return a const reference to `p` to avoid useless copies.
+
+    \note This peculiar requirement is necessary because some CGAL structures such as triangulations
+    internally manipulate points whose type might `Point_2` or `Weighted_point_2`.
```

Comment:
```suggestion
    \note This peculiar requirement is necessary because some \cgal structures such as triangulations
    internally manipulate points whose type might be `Point_2` or `Weighted_point_2`.
```